tools/check_format.py

```diff
@@ -26,7 +26,9 @@
 # Files matching these exact names can reference prod time. These include the class
 # definitions for prod time, the construction of them in main(), and perf annotation.
 # For now it includes the validation server but that really should be injected too.
-PROD_TIME_WHITELIST = ('./source/common/common/utility.h',
+REAL_TIME_WHITELIST = ('./source/common/common/utility.h',
```

Comment:
Can you update the comment above to consistently refer to this as "real clock time" or whatever sounds good? It still calls it "prod time" in the comment.

Reply:
done